### PR TITLE
Corporation fixes

### PR DESCRIPTION
**Pull request by MeuhV1**
Fixed wrong buttons in corporation advisor
Fixed corpotion pedia page when playing Brazil

REQUIRES RECOMPILING THE DLL

### CODE REVIEW

**Review by dguenms — CHANGES_REQUESTED** [review]
I don't think I want the Great People lists to be visible in the civilopedia. Discovering all the names that exist should be part of the game, and a reward for getting those GPs, in my opinion.

Can you separate the actual bug fixes from this pull request so I can approve them on their own? Everything looks fine there for me.

**Comment by MeuhV1** [comment]
Done

**Comment by MeuhV1** [comment]
Done

File: Assets/Python/Screens/CvCorporationScreen.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ def __init__(self):
 		self.W_INFLUENCES_AREA = self.X_CITY2_AREA + self.W_CITY_AREA - self.X_INFLUENCES_AREA
 		self.H_INFLUENCES_AREA = self.H_REQUIREMENTS_AREA
 		self.INFLUENCES_ID =  "InfluencessAreaWidget"
+		self.iCorporationTech = iCorporation
```
Comment (dguenms) [inline Assets/Python/Screens/CvCorporationScreen.py]: I suggest to solve this using
```
from Consts import *
from Consts import iCorporation as iCorporationTech
```

Should work iirc. At least the iCorporationTech shouldn't be a member variable of the CvCorporationScreen class.

**Review by dguenms — CHANGES_REQUESTED** [review]

**Review by dguenms — APPROVED** [review]

**Comment by MeuhV1** [comment]
Incorporated (pun intended) suggestions.

**Pull request merged**